env-defaults

```diff
@@ -269,6 +269,13 @@ HARMONY_GDAL_ADAPTER_LIMITS_CPU=128m
 HARMONY_GDAL_ADAPTER_LIMITS_MEMORY=8Gi
 HARMONY_GDAL_ADAPTER_INVOCATION_ARGS='python -m gdal_subsetter'
 
+HYBIG_IMAGE=sds/harmony-browse-image-generator:latest
+HYBIG_REQUESTS_CPU=128m
+HYBIG_REQUESTS_MEMORY=128Mi
+HYBIG_LIMITS_CPU=128m
```

Comment:
Note to ourselves that I don't think LIMITS_CPU parameters are being used anywhere, but if they were we wouldn't want to limit services to 1/8 a VCPU.